go-client/client/example_table_ops_test.go

```diff
@@ -8,24 +8,92 @@ import (
+	//   col[1][Close]: [53.8 88.5 38.7 453 26.7 544.9 13.4]
+	//   col[2][Volume]: [87000 6060842 138000 138000000 19000 48300 1500]
+	//
+	// New data:
+	// record:
+	//   schema:
+	//   fields: 3
+	//     - Ticker: type=utf8, nullable
+	//         metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "java.lang.String", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//     - Close: type=float32, nullable
+	//        metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "float", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//     - Volume: type=int32, nullable
+	//         metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "int", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//   metadata: ["deephaven:attribute_type.AddOnly": "java.lang.Boolean", "deephaven:attribute.AddOnly": "true"]
+	//   rows: 5
+	//   col[0][Ticker]: ["XRX" "IBM" "GME" "AAPL" "ZNGA"]
+	//   col[1][Close]: [53.8 38.7 453 26.7 544.9]
+	//   col[2][Volume]: [87000 138000 138000000 19000 48300]
+	//
+	// record:
+	//   schema:
+	//   fields: 4
+	//     - Ticker: type=utf8, nullable
+	//         metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "java.lang.String", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//     - Close: type=float32, nullable
+	//        metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "float", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//     - Volume: type=int32, nullable
+	//         metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "int", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//     - Magnitude: type=int32, nullable
+	//            metadata: ["deephaven:isRowStyle": "false", "deephaven:type": "int", "deephaven:isNumberFormat": "false", "deephaven:isStyle": "false", "deephaven:isRollupColumn": "false", "deephaven:isDateFormat": "false"]
+	//   rows: 5
+	//   col[0][Ticker]: ["XRX" "IBM" "GME" "AAPL" "ZNGA"]
+	//   col[1][Close]: [53.8 38.7 453 26.7 544.9]
+	//   col[2][Volume]: [87000 138000 138000000 19000 48300]
+	//   col[3][Magnitude]: [10000 100000 100000000 10000 10000]
+}
+
+// This function demonstrates how to use normal table operations.
+//
+// Normal table operations take in TableHandles as inputs, and return a TableHandle (or an error) as an output.
+// They allow for more fine-grained error handling and debugging than queries, at the cost of being more verbose.
+func doNormalOps() (string, error) {
```

Comment:
We may want a name for these other than "normal", because that is going to bias users to stay on the "normal" path.  @kosak @jcferretti Ideas?